homeassistant/components/pglab/config_flow.py

```diff
@@ -0,0 +1,66 @@
+"""Config flow for Tasmota."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant import config_entries
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.service_info.mqtt import MqttServiceInfo, ReceivePayloadType
+
+from .const import CONF_DISCOVERY_PREFIX, DISCOVERY_TOPIC, DOMAIN
+
+
+class FlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow."""
+
+    VERSION = 1
+
+    def __init__(self) -> None:
+        """Initialize flow."""
+        self._prefix = DISCOVERY_TOPIC
```

Comment:
There's no need to store this in the class because it's never used

Reply:
See previous comment.